examples/image-to-text/run_pipeline.py

```diff
@@ -149,12 +154,66 @@ def main():
 
         htcore.hpu_set_env()
 
-    generator = pipeline(
-        "image-to-text",
-        model=args.model_name_or_path,
-        torch_dtype=model_dtype,
-        device="hpu",
-    )
+    world_size = int(os.getenv("WORLD_SIZE", "1"))
+    rank = int(os.getenv("RANK", "0"))
+
+    if world_size > 1:
+        import deepspeed
+        from habana_frameworks.torch.distributed.hccl import initialize_distributed_hpu
+
```

Comment:
```suggestion
        os.environ.setdefault("PT_HPU_ENABLE_LAZY_COLLECTIVES", "true")

```
We can add this as a default for world_size > 1 and using deepspeed, or we should provide a hint in the README for user